### PR TITLE
byte array support for .Net Driver

### PR DESCRIPTION
**Pull request by praveenag**

### CODE REVIEW

File: Neo4j.Driver/Neo4j.Driver.IntegrationTests/DirectDriver/DriverIT.cs [hunk 1]
```diff
@@ -34,6 +36,49 @@ public DriverIT(ITestOutputHelper output, StandAloneIntegrationTestFixture fixtu
         {
         }
 
+        [RequireServerVersionGreaterThanOrEqualToFact("3.2.0")]
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver.IntegrationTests/DirectDriver/DriverIT.cs]: Can change all `RequireXXServerFact` to this format instead

File: Neo4j.Driver/Neo4j.Driver.IntegrationTests/Internals/IntegrationTestAttribute.cs [hunk 2]
```diff
@@ -51,6 +51,60 @@ public Require31ServerFactAttribute()
     }
 
     /// <summary>
+    /// Use `Require32ServerFact` tag for the tests that require a server with version equals to or greater than 3.2
+    /// </summary>
+    public class Require32ServerFactAttribute : FactAttribute
```
Comment (zhenlineo) [inline Neo4j.Driver/Neo4j.Driver.IntegrationTests/Internals/IntegrationTestAttribute.cs]: Can be removed

**Comment by praveenag** [comment]
yo @zhenlineo how about now?

**Comment by zhenlineo** [comment]
@praveenag Thanks a lot!
A minor naming that `RequireServerVersionGreaterThanOrEqualToFactAttribute` can be simplified as `RequireServerVersionGreaterThanOrEqualToFact`.
Besides that, all looks good.

**Comment by praveenag** [comment]
@zhenlineo Now?

**Pull request merged**